docs/src/assets/chainrules.css

```diff
@@ -0,0 +1,40 @@
+/* TOC */
+nav.toc {
+	background-color: #FFEC8B;
+	box-shadow: inset -14px 0px 5px -12px rgb(210,210,210);
+}
+
+nav.toc ul.internal {
```

Comment:
It's been a while since I've CSS'd, I think you wanna do something like:

```
nav.toc ul.internal {
    background-color: inherit;
    box-shadow: inherit;
    list-style: none;
}
```

Reply:
maybe for `box-shadow`, not for `backgrond-color`
which I am setting to something different.